Sources/UIKitExtensions/UISwitch+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UISwitchFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import UIKit
+public extension ConfigurationSet where Base: UISwitch {
+    
+    func setOn(_ on: Bool, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UISwitch: Base ) in
+            UISwitch.setOn(on, animated: animated)
```

Comment:
This function is only useful for existing UISwitch whose isOn change should be animated.
The isOn Property ist writable directly so there is no need for this function.